### PR TITLE
ENH: add EL3052 to TM2K2 and AT1K2

### PR DESCRIPTION
**Pull request by nrwslac**
## Description

ENH: add EL3052 to TM2K2 and AT1K2
## Pre-merge checklist
- [ ] Code works interactively
- [ ] Code contains descriptive comments
- [ ] Test suite passes locally
- [ ] Libraries are set to fixed versions and not ``Always Newest``
- [ ] Code committed with ``pre-commit`` (alternatively ``pre-commit run --all-files``)

### CODE REVIEW

File: plc-kfe-rix-motion/_Config/IO/PLC Rail (EtherCAT)/Power (EK1200)/Fiber Coupler (EK1521-0010)/Term 123 (EK1100)/Term 140 (EK1122)/TM2K2-K2B (EK1100).xti [hunk 1]
```diff
@@ -1,5 +1,5 @@
-<?xml version="1.0"?>
-<TcSmItem xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://www.beckhoff.com/schemas/2012/07/TcSmProject" TcSmVersion="1.0" TcVersion="3.1.4024.4" ClassName="CFlbTermDef" SubType="9099">
```
Comment (nrwslac) [inline plc-kfe-rix-motion/_Config/IO/PLC Rail (EtherCAT)/Power (EK1200)/Fiber Coupler (EK1521-0010)/Term 123 (EK1100)/Term 140 (EK1122)/TM2K2-K2B (EK1100).xti:2]: looks like I should rebuild with 4024

Comment (ZLLentz) [reply plc-kfe-rix-motion/_Config/IO/PLC Rail (EtherCAT)/Power (EK1200)/Fiber Coupler (EK1521-0010)/Term 123 (EK1100)/Term 140 (EK1122)/TM2K2-K2B (EK1100).xti:2]: The PLC here runs on 4022, so I don't think you should worry about it.

**Review by ZLLentz — APPROVED** [review]
This looks reasonable to me
When I merge #58 on top of this I will resolve the tmc conflict by blowing away all changes and rebuilding.

File: plc-kfe-rix-motion/_Config/IO/PLC Rail (EtherCAT)/Power (EK1200)/Fiber Coupler (EK1521-0010)/Term 123 (EK1100)/Term 140 (EK1122)/TM2K2-K2B (EK1100).xti (hunk 1 above)
Comment (nrwslac) [reply plc-kfe-rix-motion/_Config/IO/PLC Rail (EtherCAT)/Power (EK1200)/Fiber Coupler (EK1521-0010)/Term 123 (EK1100)/Term 140 (EK1122)/TM2K2-K2B (EK1100).xti:2]: Okay. @jyotiphy also informed me not to build this proj with 4024.

**Pull request merged**